source/standards/production-readiness.html.md.erb

```diff
@@ -1,28 +1,50 @@
 ---
-title: How to know when your service is ready for production
+title: How to check if your service is production ready
 expires: 2019-03-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-Before you take a service into production, and introduce it to real users, you should make sure that it meets the technical standards set by GDS and your team. Some of these standards are provided in the GDS Way but your team may have set its own standards. For example, your team may require that all new services must be connected to logit before they go to production. If you neglect to implement a standard before users start using your service then you could have difficulties operating your service or supporting it when a fault occurs.
+Before taking your service into production and introducing it to live users make sure it meets the relevant technical standards set by GDS and your team.
 
-You can use a checklist to confirm whether your service is ready for production. Checks in the list should reference a standard that must be in place before you can take any service into production. You can also include communication checks to ensure that specific groups or individuals are informed in advance.
+Some of these standards are provided in the GDS Way but your team may also have its own specific standards. For example, your team may require that all its new services are connected to [Logit][] before going to production.
 
-The checklist should be reusable so that your team can apply it every time you wish to take a new service, or subcomponent, into production. 
+If you do not implement a standard before people start using your service then you could have difficulties operating your service or supporting it when a fault occurs. **For example…**
 
-## Example
+## Use a checklist
 
-```
-In order to take a service into production:
-- It MUST be possible to search logs 
-- It MUST be possible to monitor the health of the service
-- It MUST be possible to view application metrics
-- It MUST be possible to be alerted when the service has problems
-- It MUST be possible to follow a documented deployment process
-- You MUST have agreement to proceed from IA
-- You MUST inform PERSON A
-- You MUST inform GROUP B
-```
+To avoid problems when your service goes into production you should use a checklist to confirm whether your service is ready. Each check should reference a standard that's already in place before you can take any service into production. **For example…**
```

Comment:
It would make this stronger if we can give an example here and in the next paragraph.